fsthttp/backend.go

```diff
@@ -311,6 +311,52 @@ func (b *BackendOptions) PoolConnections(poolingOn bool) *BackendOptions {
 	return b
 }
 
+// HttpKeepaliveTime configures how long to allow HTTP connections to remain
+// idle in a connection pool before it should be considered closed.
+func (b *BackendOptions) HttpKeepaliveTime(time time.Duration) *BackendOptions {
```

Comment:
Do we want `HTTP` instead of `Http` ?

Reply:
https://pkg.go.dev/net/http#ParseHTTPVersion says yes...?

Reply:
Fixed in 107ec31!